README.md

```diff
@@ -126,3 +126,9 @@ Here is a base chain template with every feature the configuration accepts. Acco
     }
 },
 ```
+
+---
```

Comment:
This should be in the install section near the top.

Like:

Install on Mac / Linux
... current install

Install on Windows
... this line